UXO_2018_IREC.tex

```diff
@@ -465,4 +466,69 @@ \subsection{[A5] Abort Procedure - Remote Disconnect or Ignition Failure}
 \setcounter{checklistnum}{0}
 % end A5 section -- }}}
 
+% begin abort procedure gamma -- {{{
+\subsection{[A{\LARGE$\gamma$}] Abort Procedure - Voice Contact Loss - For Launch Control Operators}
+\begin{checklist}
+    \item \control{}: Remove the system control key from the client side box.
+    \item \ops{}: Attempt to regain communication with the operators at the pad:
+    \begin{checklist}
+        \item Send ``\ops{} to \secondary{}, \ops{} to \secondary{}, \secondary{} please come in''.
+        \begin{checklist}[label=$\bullet$]
+            \item If contact is restored:
+            \begin{checklist}
+                \item Return to normal operations.
+            \end{checklist}
+        \end{checklist}
+        \item Check batteries in radio.
+        \item Check that radio is set to the proper channel.
+        \item Check that radio volume is high enough.
+        \item Wait 30 seconds, then send message again.
+        \begin{checklist}[label=$\bullet$]
+            \item If contact is restored:
+            \begin{checklist}
+                \item Return to normal operations.
+            \end{checklist}
+        \end{checklist}
+    \end{checklist}
+    \item \ops: Wait 30 seconds.
+    \item \ops: Send ``\ops{} to \secondary{}, \ops{} to \secondary{}, going to full abort. Say again, going to full abort.
+    \item \ops: Inform the ESRA official that we have aborted.

```

Comment:
"Inform the Launch Control Officer that launch operations will be aborted."

mostly I don't like using "we"